src/main/java/us/warframestat/moddetection/api/utils/WarframeMarketAPI.java

```diff
@@ -105,7 +105,6 @@ public static int getPrice(String item, String platform) {
         // check for rate limit, if so, wait and try again
         try {
           if (connection.getResponseCode() == 503) {
-            logger.info("RIP: Rate limited. Retrying in 5 seconds.");
```

Comment:
Logging here would still be useful, even if lower priority.  No-one wants to wait with no output on what's going on

Reply:
yeah, maybe do debug here instead of info?